packages/components/src/components/post-icon/readme.md

```diff
@@ -0,0 +1,23 @@
+# post-icon
```

Comment:
This file should not be pushed : the component's readme.md are now part of .gitignore

Reply:
I'm not sure if we should not track the readme files. It's possible to add custom descriptions on top of the generated table and readme files are automatically displayed on github, for example here: [post-icon](https://github.com/swisspost/design-system/tree/feat-initialize-icons-package/packages/components/src/components/post-icon). WDYT?

Reply:
Would push it, even if it's completely auto-generated.

Reply:
Removed the line in the .gitignore, readmes are now tracked in git.

Reply:
Sure, we can push them too.
Could you please remove the path from .gitignore then?